### PR TITLE
Finished my basic working obstacle course pages

### PR DESCRIPTION
**Pull request by amirajas**
I also linked to the pages that followed in them to continue the game

### CODE REVIEW

File: server/www/decoybutton.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+<button type="button" class="btn btn-success">Download Ticket</button>
+<button type="button" class="btn btn-danger">Click here to download</button>
+<button type="button" class="btn btn-warning">CLICK HERE</button>
+<button type="button" class="btn btn-info">CONFIRM TICKET</button>
+<button type="button" class="btn btn-light">CLICK HERE NOW</button>
+<a href = "https://www.darkpatterns.org/types-of-dark-pattern"> <button type="button" class="btn btn-dark">Accept Purchase</button></a>
+ <br><br><br>
+<button type="button" class="btn btn-link">CLICK TO DOWNLOAD</button>
+<br><br>
+<a href="https://www.nytimes.com/2012/02/19/magazine/shopping-habits.html?pagewanted=all"><button type="button" class="btn btn-primary btn-lg btn-block">CLICK HERE FOR A FREE TICKET</button></a>
```
Comment (nbriz) [inline server/www/decoybutton.html]: just a thought (but feel free to disagree), but what if these links were set to open in new tabs (by adding the `target="_blank"` attribute to the `a` tags). this way u don't "leave" the game, but rather take more of an "aside"

File: server/www/flightinsure.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+    <meta charset="utf-8">
+    <title>Flight Insurance</title>
+  </head>
+  <body>
+    <h3>Would you like to add insurance to your flight?</h3>
+    <script type="text/javascript" src="js/bootstrap.bundle.js"></script>
+    <script type="text/javascript" src="js/bootstrap.min.js"></script>
```
Comment (nbriz) [inline server/www/flightinsure.html]: these bootstrap files aren't in the repo. they might be in ur local computer, but haven't been committed so they aren't included in this pull request && thus don't actually load when i test. did u mean to leave these script tags here? if so u should also commit those bootstrap files, if not u should remove these

**Review by nbriz — CHANGES_REQUESTED** [review]

File: server/www/decoybutton.html (hunk 1 above)
Comment (amirajas) [reply server/www/decoybutton.html]: I think that's a good idea, I'll switch it in my code.

File: server/www/flightinsure.html (hunk 2 above)
Comment (amirajas) [reply server/www/flightinsure.html]: I think I meant to remove them but I will check

Comment (amirajas) [reply server/www/flightinsure.html]: Oh that page was suppose to get deleted, sorry about that

Comment (nbriz) [reply server/www/flightinsure.html]: ok cool, merging now

**Pull request merged**